rgd/utility.py

```diff
@@ -64,6 +69,11 @@ def parse(self, stream, media_type=None, parser_context=None):
             if isinstance(getattr(model, key), fields.related_descriptors.ManyToManyDescriptor):
                 for val in value.split(','):
                     qdict.update({key: val.strip('"')})
+            elif model.__name__ == 'RasterEntry' and key in raster_arrays:
```

Comment:
We really don't want to leak our models into the general utility function.  It will cause future maintenance problems.  If django-rest-framework-gis is added to the installed apps, is this still necessary?

Reply:
So overall, all of the geometry fields are already being serialized to geojson from what I can tell looking at my sample queries, but these raster_array fields are from `django.contrib.postgres`  so installing django-rest-framework-gis didn't seem to help.

Reply:
Did a simple fix to not leak models